lib/CachedSource.js

```diff
@@ -56,12 +58,18 @@ class CachedSource extends Source {
 		}
 		const bufferedMaps = new Map();
 		for (const pair of this._cachedMaps) {
-			if (pair[1].bufferedMap === undefined) {
-				pair[1].bufferedMap = mapToBufferedMap(pair[1].map);
+			let cacheEntry = pair[1];
+			if (cacheEntry.bufferedMap === undefined) {
+				if (cacheEntry.map === undefined) {
+					continue;
```

Comment:
I am not certain what approach would be best here. In its current form, this means that `CachedSource`s which have only been accessed using `node` will have nothing to cache, requiring the map or node to be recomputed when loaded from the cache.

This _could_ convert the `cacheEntry.node` (if it exists) into a map and then serialize that. When the `CachedSource` is then reloaded from disk, accessing `node` can leverage the serialised map instead of delegating to the original source. However, this would still require a `SourceMapConsumer` which I'm not sure would actually be more efficient than just delegating to the original source's `node` property (assuming it exists).